micro-ui/web/micro-ui-internals/packages/svg-components/src/svg/GlobeLocationPin.js

```
@@ -0,0 +1,43 @@
+export const GlobeLocationPin = ({ className, height = "40", width = "40", style = {}, fill = COLOR_FILL, onClick = null }) => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width={width}
+      height={height}
+      viewBox="0 0 40 40"
+      fill="none"
+      className={className}
+      onClick={onClick}
+      style={style}
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)